package.json

```diff
@@ -38,8 +41,7 @@
     ]
   },
   "files": [
-    "dist",
-    "src"
```

Comment:
Can you bring back `src` folder? I think some users can rely on this and I don't see reasons break it.